vite.config.ts

```diff
@@ -0,0 +1,7 @@
+import { defineConfig } from 'vite'
+import react from '@vitejs/plugin-react'
```

Comment:
**praise**: Good job including a reference to the Vite configuration documentation. This is helpful for future developers.